e2e/util_test.go

```diff
@@ -0,0 +1,109 @@
+package e2e
+
+import (
+	"context"
+	"crypto/rand"
+	_ "embed"
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/authzed/authzed-go/v1"
+	"github.com/authzed/grpcutil"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/ory/dockertest/v3"
+	"github.com/rs/zerolog/log"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+)
+
+//go:embed fixtures/test_schema.sql
+var testSchema string
+
+func spicedb(require *require.Assertions) (*authzed.Client, func()) {
+	pool, err := dockertest.NewPool("")
+	require.NoError(err)
+
+	resource, err := pool.RunWithOptions(&dockertest.RunOptions{
+		Cmd:          strings.Split(`serve --grpc-preshared-key somerandomkeyhere --grpc-no-tls --http-no-tls`, " "),
```

Comment:
oh boy, this will change soon

Reply:
yep, but it will coordinated with a bump in the image's tag